main.cpp

```diff
@@ -54,6 +59,7 @@ struct CLICallbacks
 		callbacks[cli] = func;
 	}
 	unordered_map<string, function<void(CLIParser &)>> callbacks;
+	uint64_t pad0;
```

Comment:
The manual padding here seems very strange and makes the code uglier for no apparent gain. What would this help? The obvious concern here is that struct layout has arbitrary padding, but none of the code relies on a particular struct memory layout.